### DISCUSSION TITLE
Welcome to postgresql-to-mssql Discussions!

### DISCUSSION THREAD

**Discussion by sanamhub** [sanamhub/postgresql-to-mssql#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sybrand055** [comment]
Good day, I was trying out your program postgresql-to-mssql. I am having problems getting the PostgreSQL connection string right. I am copying the connection string out of postgres but I have no luck. (I add the password where the xxxxx are)
![image](https://github.com/sanamhub/postgresql-to-mssql/assets/151634485/10e012c0-4e3d-4b2a-8c9c-b3627cee2aab)
![image](https://github.com/sanamhub/postgresql-to-mssql/assets/151634485/6a962669-3761-45ec-b6e3-0bd677b32b71)
Thanks, will await your reply.

**Reply by sanamhub** [reply]
Please try this format and see if it works.

`"PostgreSqlConnectionString": "Server=127.0.0.1;Port=5432;Database=YourDatabase;User Id=postgres;Password=YourPassword;"`

PS: This program still needs finishing and I'm planning to include alternative (mssql-to-postgresql) migration as well. having said that, it should work pretty well on normal scenario, stay tuned.

**Reply by Sybrand055** [reply]
Thanks for you feedback much appreciated, I will try.

**Reply by Sybrand055** [reply]
Hi @sanamhub I did try the connection string as suggested but it did not work either I will try it on a different DB as well do you know if you need to add connections by adding an entry to the pg_hba.conf? This is just a local install on windows to test but that might be the issue. Will check it later and let you know. 
![image](https://github.com/sanamhub/postgresql-to-mssql/assets/151634485/c5c23aae-ceae-44c0-8113-4b93b117707c)
thanks have great day.

**I have actually come to realise I am able to connect through PGADMIN so it's not a restriction problem.**

**Reply by sanamhub** [reply]
Hi @Sybrand055, Thanks for digging.

I did try and it's working perfectly, please let me know if you're still unable.

![image](https://github.com/sanamhub/postgresql-to-mssql/assets/31341013/c6a89692-2371-438a-8ef3-12aa17886ed9)

FIY: Get updated code with using .NET 8 by pulling latest changes from `main` branch.

This is little bit messy for a while, I need to cleanup and release an executable.

Please file and issue if any.

Have a great day!

**Reply by Sybrand055** [reply]
Thanks for this will test tomorrow.

**Reply by Sybrand055** [reply]
Hi @sanamhub I was able to run the program without glitches. I had to uninstall my Postgres and reinstall I think there was something wrong with the installation and that was causing my Issues. 
Just a suggestion to make this app robust you should consider it to migrate the following as well in your app, **stored procedures, functions, packages, views, and triggers.**

Thank you so much for your help, and I think this app is a great idea, I will watch out for any future changes you might make to it and good luck,

![image](https://github.com/sanamhub/postgresql-to-mssql/assets/151634485/34dd2310-34ac-4220-b0a0-e3217e3116a5)
